assets/locales/active.en.toml

```
@@ -364,9 +364,9 @@ description = "Try the new way to filter and download this data:"
 one = "Try the new way to filter and download this data:"
 
 [DownloadOption]
```

It might be worth considering changing this dictionary tag to be closer to the new wording as it can be easier to find the bit in the template if you can search a keyword.

Something like the following might work:
```suggestion
[EditionsInDataset]
```